applicationinsights.dev.json

```diff
@@ -16,32 +16,38 @@
   "sampling": {
     "percentage": 100,
     "overrides": [
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "server.address",
+            "value": ".+",
```

Comment:
same approach here as for prisoner search - want to still sample InProc Publish messages at 100%, but all other dependency that have a `server.address` at a lower rate